### PR TITLE
Zb fix imagemagick

### PR DESCRIPTION
**Pull request by boatrite**

### CODE REVIEW

File: ansible/roles/rails/tasks/main.yml [hunk 1]
```diff
@@ -4,16 +4,31 @@
...
+      name: ['imagemagick', 'libmagickwand-dev']
+    become: true
+    tags:
+      - imagemagick
+
+  - name: Update ImageMagick policy to enable reading PDF files
+    lineinfile:
+      path: /etc/ImageMagick-6/policy.xml
+      regexp: '<policy domain="coder" rights="none" pattern="PDF" />'
+      line: '  <policy domain="coder" rights="read" pattern="PDF" />'
```
Comment (samsonasu) [inline ansible/roles/rails/tasks/main.yml]: got some extra space here

File: lib/subspace/commands/provision.rb [hunk 2]
```diff
@@ -1,5 +1,5 @@
 class Subspace::Commands::Provision < Subspace::Commands::Base
-  PASS_THROUGH_PARAMS = ["tags", "start-at-task"]
+  PASS_THROUGH_PARAMS = ["tags", "start-at-task", "private-key"]
```
Comment (samsonasu) [inline lib/subspace/commands/provision.rb]: What does this do?

**Comment by samsonasu** [comment]
version bump?

File: lib/subspace/commands/provision.rb (hunk 2 above)
Comment (boatrite) [reply lib/subspace/commands/provision.rb]: Make it possible to specify private key to use when running playbook

**Pull request merged**